src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -577,10 +577,10 @@ class AutoRerouteSpec extends HealthCheckSpecification {
         def blockData = switchHelper.knockoutSwitch(switchToManipulate, RW)
         def isSwitchActivated = false
         wait(WAIT_OFFSET) {
-            def prevHistorySize = northbound.getFlowHistory(flow.flowId).size()
+            def prevHistorySize = flowHelper.getHistorySize(flow.flowId)
             Wrappers.timedLoop(4) {
```

Comment:
minor: Could you please use the same approach for the timedLoop as now we have two  Wrappers.timedLoop and timedLoop(line 529)?
It is an old part of code, but do you know why we use  hardcoded 4 value (Wrappers.timedLoop(4))?

Reply:
Added static imports for 'Wrappers' methods. 
Don't know why 4, guess it's just made empirically, to make test stable but not too long.